### PR TITLE
WIP: debugging ms5611 on spi2

### PR DESCRIPTION
**Pull request by kd0aij**
still reading all zeroes from ms5611 prom with spi2 clock set to 1MHz

### CODE REVIEW

**Comment by kd0aij** [comment]
tested with spi2 clock at 1MHz prior to cleanup
need to rebase to single commit and retest

**Comment by kd0aij** [comment]
tested on production board at 12MHz; 
@LorenzMeier ready for you to finish up

File: src/drivers/ms5611/ms5611_nuttx.cpp [hunk 1]
```diff
@@ -829,7 +829,7 @@ struct ms5611_bus_option {
 	{ MS5611_BUS_SPI_EXTERNAL, "/dev/ms5611_spi_ext", &MS5611_spi_interface, PX4_SPI_BUS_EXT, NULL },
 #endif
 #ifdef PX4_SPIDEV_BARO
-	{ MS5611_BUS_SPI_INTERNAL, "/dev/ms5611_spi_int", &MS5611_spi_interface, PX4_SPI_BUS_SENSORS, NULL },
+	{ MS5611_BUS_SPI_INTERNAL, "/dev/ms5611_spi_int", &MS5611_spi_interface, PX4_SPI_BUS_RAMTRON, NULL },
```
Comment (LorenzMeier) [inline src/drivers/ms5611/ms5611_nuttx.cpp]: This one is not correct, but I'll fix it.

**Comment by LorenzMeier** [comment]
Replaced with https://github.com/PX4/Firmware/pull/3416

**Pull request closed without merging**